### PR TITLE
#1323

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/Attributes.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+	static {
+		Field[] fields = Attributes.class.getFields();
+		for (Field field : fields) {
+			try {
+				if (!field.isSynthetic() && field.getType() == String.class) {
+					set.add((String)field.get(null));
+				}
+			} catch (IllegalArgumentException | IllegalAccessException ex) {
+				// don't care
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/Attributes.java]: When would this happen?

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 2]
```diff
@@ -132,4 +94,110 @@ public Response instantiateTemplate(@Context HttpServletRequest request, @PathPa
...
+		Map<String, Item> childMap = makeChildItemMap(childItems);
+		Iterator<JsonElement> iterator = children.iterator();
+
+		while (iterator.hasNext()) {
+			
+			String attr = "";
+			Layer layer = null;
+
+			JsonObject child = iterator.next().getAsJsonObject();
+			JsonElement childId = child.get("id");
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Did you not want to make ENUMs for this?

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/Attributes.java (hunk 1 above)
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/Attributes.java]: Should be never, but they are checked exceptions that I had to deal with.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java (hunk 2 above)
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Nah, I could probably pull them out to constants, but not sure it is worth it really.

Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Agreed

File: coastal-hazards-portal/src/main/resources/logback.xml [hunk 3]
```diff
@@ -20,7 +20,7 @@
 		</timeBasedFileNamingAndTriggeringPolicy>
 
 	</appender>
-	<root level="TRACE">
+	<root level="DEBUG">
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/resources/logback.xml]: :+1:

**Comment by jiwalker-usgs** [comment]
Sorry for all the reformatting btw.  I was refactoring the SLD classes and reformatted them along with it.

**Comment by isuftin** [comment]
:ship: :it:

**Comment by jiwalker-usgs** [comment]
:hehehe:

**Pull request merged**